### PR TITLE
Microsoft Entra How-To Guide

### PR DESCRIPTION
**Pull request by Jesse-Ben**
## This PR edits our docs site to include instructions on deploying SSO with Palette and Microsoft Entra ID (formerly Azure Active Directory) using OpenID Connect (OIDC). 

The instructions assist the user with SSO integration for Palette and K8s clusters.

This PR .....

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-489)

### CODE REVIEW

File: docs/docs-content/user-management/saml-sso/saml-sso.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ Enable SSO by following our [Enable SSO in Palette](enable-saml.md) guide.
 - [Palette SSO with Okta OIDC](palette-sso-with-okta.md)
 
 
+- [Palette SSO with Microsoft Entra ID OIDC](palette-sso-with-entra-id.md)
```
[the remark this answers is not in the document]
Comment (Jesse-Ben) [reply docs/docs-content/user-management/saml-sso/saml-sso.md:38]: Should be "Entra".

File: docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+tags: ["user-management", "oidc-sso", "entra-id", "azure-ad"]
+---
+Single sign-on (SSO) is an authentication method that enables users to log in to multiple applications and websites with one set of credentials. SSO uses certificates to establish and maintain trust between the service provider (SP) and an identity provider. 
+
+This document will guide you with end-to-end steps for deploying and enabling SSO with Microsoft Entra based on OpenID Connect for integration with Palette and Kubernetes clusters.
+
+## Prerequisites
+- Spectro Cloud Palette version 4.0.X or greater. 
+
+- [Microsoft Entra ID](https://entra.microsoft.com/#home) subscription. You will need an account with one of the following roles: Global Administrator, Cloud Application Administrator, Application Administrator, or owner of the service principal.
```
[the remark this answers is not in the document]
Comment (Jesse-Ben) [reply docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md:18]: yes.

File: docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+Single sign-on (SSO) is an authentication method that enables users to log in to multiple applications and websites with one set of credentials. SSO uses certificates to establish and maintain trust between the service provider (SP) and an identity provider. 
+
+This document will guide you with end-to-end steps for deploying and enabling SSO with Microsoft Entra based on OpenID Connect for integration with Palette and Kubernetes clusters.
+
+## Prerequisites
+- Spectro Cloud Palette version 4.0.X or greater. 
+
+- [Microsoft Entra ID](https://entra.microsoft.com/#home) subscription. You will need an account with one of the following roles: Global Administrator, Cloud Application Administrator, Application Administrator, or owner of the service principal. 
+
+- If you want to use the same Microsoft Entra ID application for OIDC-based SSO into your Kubernetes cluster itself, you need to install [kubelogin](https://github.com/int128/kubelogin) on your local workstation to handle retrieval of access tokens for your cluster.
```
[the remark this answers is not in the document]
Comment (Jesse-Ben) [reply docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md:20]: yes

[the remark this answers is not in the document]
Comment (Jesse-Ben) [reply docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md:20]: yes

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Hi @Jesse-Ben thanks for the contribution. A couple of things that needs to be addressed first. The image count is a bit too high, we aim for a max of 12 images per page. The more images, the more of an impact it has on the load time. Let's reduce the count to 12 please.

The other issue is that the builds are not completing. Most likely it's the usage of custom HTML elements, please remove these. 

When guiding the user through pages, we don't use arrows ->. Please write out in a complete sentence where to click. For example, "Select Tenant Settings on the left Main Menu. From the Tenant Menu, select OIDC."

**Comment by karl-cardenas-coding** [comment]
Closed per @Jesse-Ben 's guidance. PR #1966 carries over this work

**Pull request closed without merging**